docker/Dockerfile

```diff
@@ -0,0 +1,36 @@
+# First stage: build the application
+FROM golang:1.13.0-alpine AS builder
+ARG GOARCH
+ARG GOARM
+
+WORKDIR /go/src/github.com/CESARBR/knot-babeltower
+COPY . .
+
+# Install build utilities and glide dependency manager
+RUN apk add --update --no-cache --virtual .build-deps \
+    make \
+    git \
+    curl
```

Comment:
Will we be needing `curl`, even if we don't use glide? :smile: 
If not, we can remove it!

Reply:
Done. Thank you for suggesting a security improvement.